derived_images/yarn/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM opensuse/amd64:42.3
+MAINTAINER SUSE Containers Team <containers@suse.com>
+
+RUN mkdir -m 0600 /tmp/build && \
+    gpg --homedir /tmp/build --keyserver ha.pool.sks-keyservers.net --recv-keys 55A0B34D49501BB7CA474F5AA193FBB572174FC2 && \
+    gpg --homedir /tmp/build --export --armor 55A0B34D49501BB7CA474F5AA193FBB572174FC2 > /tmp/build/repo.key && \
+    rpm --import /tmp/build/repo.key && \
+    rm -rf /tmp/build && \
+    # Now add the repository and install yarn.
+    zypper ar -f obs://Virtualization:containers:Portus/openSUSE_Leap_42.3 portus && \
```

Comment:
Do we need portus stuff here?

Reply:
Yes, because yarn is not yet on `devel:languages:nodejs`, and I'm honestly a bit afraid of submitting the package there. So I simply decided to pick `yarn` and `nodejs6` from `Virtualization:containers:Portus` instead.

Reply:
I think so because the yarn package is taken from this repo